library/public/loader.h

```diff
@@ -61,6 +61,16 @@ class F3D_EXPORT loader
    */
   virtual loader& loadGeometry(const std::string& filePath, bool reset = false) = 0;
 
+  /**
+   * Load a geometry from memory buffers.
+   * Reset the scene before loading if a full scene was loaded previously or if reset is set to
+   * false, do not reset if only loaded geometries previously. Geometries loader using this method
+   * will be available in a default scene and use all default scene related options. Throw a
+   * load_failure_exception on failure.
```

Comment:
Lets improve the doc, what I've written is unclear and also false at times.

```
   * Calling this method will reset the scene before loading if a full scene was loaded previously or if the reset argument is set to true, It will not reset if only geometries were loaded previously. 
   * Geometries loaded using this method will be available in a default scene and use all default scene related options. 
   * Throw a load_failure_exception on failure.
   ```
   
   Update the other method doc as well.